snippets/csharp/VS_Snippets_Remoting/SurrogateSelector/CS/SurrogateSelector.cs

```diff
@@ -34,7 +34,7 @@ sealed class EmployeeSerializationSurrogate : ISerializationSurrogate
         info.AddValue("address", emp.address);
     }
 
-    // Deserialize the Employee object to set the object�s name and address fields.
+    // Deserialize the Employee object to set the object�s name and address fields.
```

Comment:
I can't see the symbol for the quotes here, maybe just leave these docs lines as they were?